### PR TITLE
adding post on anna maria sichani

### PR DESCRIPTION
**Pull request by arojascastro**
Please review the text announcing @amsichani and approve my committ.

### CODE REVIEW

**Comment by arojascastro** [comment]
Well, maybe this post needs a picture of Anna-Maria.

**Comment by arojascastro** [comment]
Travis was complaining about the field `author`, I changed it to `authors` but still is not working.

Now the message I get is: `The command "_build/build.sh" exited with 1.`. Any idea?

By the way in the wiki the metadata says `author` instead of `authors`: https://github.com/programminghistorian/jekyll/wiki/Creating-Blog-Posts

Should we change that?

**Comment by acrymble** [comment]
ya if it's wrong, change it.

File: _posts/2018-05-24-anna-maria-sichani.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Anna-Maria Sichani joins Project Team
-author: Antonio Rojas Castro
+authors: Antonio Rojas Castro
```
Comment (walshbr) [inline _posts/2018-05-24-anna-maria-sichani.md]: This should be in this format, even if you only have one author for the post.
```
authors:
- Maria José Afanador-Llach
- Adam Crymble
```

**Comment by walshbr** [comment]
Not sure if I correctly commented on the code in a way that will show up for you @arojascastro, but the authors field should look like this even if you only have one author:

```
authors:
- Maria José Afanador-Llach
- Adam Crymble
```
So it just needs a new line and a dash before your name.

**Comment by walshbr** [comment]
I think the remaining 404s are going to be fixed by #859 when that is merged @arojascastro but @mdlincoln can confirm

**Comment by mdlincoln** [comment]
@walshbr correct

**Review by mdlincoln — APPROVED** [review]
This reads great to me. Merge it whenever you're ready @arojascastro

**Pull request merged**